### DISCUSSION TITLE
2nd job label is a "?"

### DISCUSSION THREAD

**Discussion by allischalmersman** [swar/Swar-Chia-Plot-Manager#170] [Q&A] [answered]
I'm sure I'm being stupid again, but does anybody have any idea why my 2nd job name is being recorded as a question mark?  

![image](https://user-images.githubusercontent.com/83618954/117557194-8756b680-b03e-11eb-8f27-a1651cb196fd.png)

![image](https://user-images.githubusercontent.com/83618954/117557226-c7b63480-b03e-11eb-86c0-6f9ce58c901b.png)

**Comment by allischalmersman** [comment]
I also just noticed that the R: drive isn't showing in the manager viewer which is what the "?" job is using as a temp drive.  I would assume this is related

**Comment by allischalmersman** [comment, marked as answer]
The seems to be resolved after restarting the Viewer.  I had restarted the manager but not the viewer after updating the config file.  Sill getting a ? on the "plots" section of the NAS but thats discussed in another thread.

**Comment by aresstokrat** [comment]
have same issue, restarting view does't help

**Reply by hanuxxx** [reply]
close the view window and open a new, solved.
